src/modules/menu-dishes/menu-dishes.service.ts

```diff
@@ -1,14 +1,22 @@
-import { ConflictException, Injectable } from '@nestjs/common';
+import {
+  ConflictException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
 import { plainToClass } from 'class-transformer';
 import { MenuDishesCreateDto } from './dtos/menu-dishes-create.dto';
 import { MenuDish } from './menu-dishes.entity';
 import { MenuDishesRepository } from './menu-dishes.repository';
 import { MenuDishesResponseDto } from './dtos/menu-dishes-response.dto';
 import { MenuDishUpdateDto } from './dtos/menu-dishes-update.dto';
+import { MenuCategoryService } from '../menu-category/menu-category.service';
 
 @Injectable()
 export class MenuDishesService {
-  constructor(private readonly _menuDishesRepository: MenuDishesRepository) {}
+  constructor(
+    private readonly _menuDishesRepository: MenuDishesRepository,
+    private readonly _menuCategoriesService: MenuCategoryService,
```

Comment:
here we should use the repositories like the other models instead use service directly to have a same conventions